### DISCUSSION TITLE
ACR is too rare, can we add a little more in game？

### DISCUSSION THREAD

**Discussion by chenpishui12345** [HasangerGames/suroi#338] [Ideas]
ACR is too rare, can we add a little more in game？

**Comment by adtcqd** [comment]
Many military factories are common in rectangular boxes.（Written with translation software.）
军工厂中有不少 在长方形的箱子里很常见

**Comment by bien-star** [comment]
ACR is one of the highest-tier weapons, which can only be found either through golden airdrops, or by opening the black flint crate, one of which spawns per game.

**Reply by Hashimk2** [reply]
Can you open the flint crate with the Maul

**Reply by bien-star** [reply]
Try it and find out

**Reply by Hashimk2** [reply]
> Can you open the flint crate with the Maul

I meant flint stone 
-_-

**Reply by kidwaih2** [reply]
> > ACR is one of the highest-tier weapons, which can only be found either through golden airdrops, or by opening the black flint _**crate**_, one of which spawns per game.
> 
> It is called a flint stone

**Comment by Steb03** [comment]
![image](https://github.com/user-attachments/assets/99d0454e-adc8-4d78-9a2d-5e31ceab912a)
ACR needs a fix either way
